cmd/ssh.go

```diff
@@ -484,7 +491,7 @@ func (cmd *SSHCmd) startServices(
 	log log.Logger,
 ) {
 	if cmd.User != "" {
-		err := tunnel.RunInContainer(
+		err := tunnel.RunServices(
```

Comment:
Could you revert this naming change please? Makes it harder to backmerge and it's not really required in this PR. 
If you absolutely want to change it, please open a new one

Reply:
The reason I changed this function's name was because it runs a server locally, so the name is misleading IMO